pandas/plotting/_core.py

```diff
@@ -2885,7 +2885,7 @@ def area(self, **kwds):
 
         Returns
         -------
-        axes : :class:`matplotlib.axes.Axes` or numpy.ndarray of them
+        axes : :class:`matplotlib.axes.Axes` or numpy.ndarray of them.
```

Comment:
```suggestion
        :class:`matplotlib.axes.Axes` or numpy.ndarray of them
```

This is the type, shouldn't finish with a full stop. If the script is warning, it's wrong (you can fix it in this PR by adding a description)